montblanc/impl/biro/v2/loaders/loaders.py

```diff
@@ -43,8 +64,9 @@ def load(self, solver, **kwargs):
         assert ms_uvw.shape == uvw_shape, \
             'MS UVW shape %s != expected %s' % (ms_uvw.shape,uvw_shape)
         uvw_rec = solver.get_array_record('uvw')
+
         uvw=np.empty(shape=uvw_rec.shape, dtype=uvw_rec.dtype)
-        uvw[:,:,1:na] = ms_uvw.reshape(ntime, nbl, 3).transpose(2,0,1) \
+        uvw[:,:,1:na] = ms_uvw.reshape(file_uvw_shape).transpose(uvw_transpose) \
```

Comment:
@marziarivi I'm not sure that the antenna positions will be assigned the correct data here. In the original change, we could expect baseline to be the last dimensions, but this is no longer the case.